public/js/admin-check-in.js

```diff
@@ -0,0 +1,45 @@
+function httpGetAsync(theUrl, callback){
+  var xmlHttp = new XMLHttpRequest();
+  xmlHttp.onreadystatechange = function() { 
+    if (xmlHttp.readyState == 4 && xmlHttp.status == 200)
```

Comment:
le sigh. hoping to add webpack + promise based library like superagent so we dont have to do this, but i'll take it

Reply:
Yea https://github.com/Bostonhacks/headstart/issues/57 
Ken and I had this discussion over messenger; possibly for next year